src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
The consumer doesn't need to understand the values. From consumer's point of view, the required data is in already mapped, documented fields of that object. The `description_values` is merely a mapping between value and in-text variable. It is undefined. The closest equivalent of that is string.Format(), which accepts any format string and any number of undefined parameters.

Your suggestion will force us to invest a lot of time in maintaining strong-typed definition of a dynamic-defined contract.